### PR TITLE
feat: create-next-app template with react-web sdk and connectkit

### PR DESCRIPTION
**Pull request by krzysu**
test with:
```
pnpm create next-app -e https://github.com/lens-protocol/lens-sdk/tree/nextjs-template/examples/lens-next-app
```

### CODE REVIEW

File: examples/lens-next-app/src/components/ConnectWalletButton.tsx [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import { ConnectKitButton } from "connectkit";
+import { Button } from "./Button";
+
+export const ConnectWalletButton = () => {
```
Comment (cesarenaldi) [inline examples/lens-next-app/src/components/ConnectWalletButton.tsx:4]: Let's keep a consistent style on these components definition. It seems NextJS docs favour Function Declarations in their examples over Arrow Function Expressions, I would stick to that style.

File: examples/lens-next-app/src/app/favicon.ico
Comment (cesarenaldi) [inline examples/lens-next-app/src/app/favicon.ico:1]: nit: we could use lens icon.

File: examples/lens-next-app/src/components/ConnectWalletButton.tsx (hunk 1 above)
Comment (krzysu) [reply examples/lens-next-app/src/components/ConnectWalletButton.tsx]: true, I missed it, fixed now!

File: examples/lens-next-app/src/app/favicon.ico
Comment (krzysu) [reply examples/lens-next-app/src/app/favicon.ico:1]: done!

**Pull request merged**